### ISSUE TITLE
Parameters with dots in their names

### ISSUE THREAD

**Issue by oladahl** [accellera-official/cci#277]
Determine if parameters with dots in their names are handled as specified in the LRM. 

If not, update the reference implementation so that it is compliant with LRM

See this mail thread for some background information:
https://workspace.accellera.org/wg/CCI-WG/mail/thread/56535

**Comment by aut0** [comment]
From Mark's E-Mail on the reflector:

I have 2 nested modules:
Module A
Module B

Module B has no params in it.
Module A declares a param called “B.c” (hence its full name will be “A.B.c”)

If you use the standard broker, A.B.c will appear, you can set it etc.

HOWEVER, the param in A will _not_ be set.

Bug or feature?

Cheers
Mark.

**Comment by markfoodyburton** [comment]
Yeah - I agree, I think this either got fixed or was never an issue.
